_posts/2020-02-12-myft-machine-learning.html

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title:  "Traversing the data mines"
+date:   2020-02-12 09:30:30 +0100
+categories: Research
+teaser: Research
+preview-image: /assets/uploads/2019/01/myft_ml_header_graph.png
+excerpt: >
+  Investigating FT user data
+---
+{% include figure.html src="/assets/uploads/2020/02/myft_ml_header_graph.png" width="80%" %}
+
+<p>Interested in the field of Machine Learning and guided by the FT's goal of adding further value through personalisation, we saw an opportunity to have a go at creating our own FT Labs content recommendation engine<sup>tm</sup>. Our early estimates warned this could be quite difficult, and we were right.</p>
```

Comment:
TM is capital, can also use HTML entity &trade; instead of <sup>

Reply:
Done